### ISSUE TITLE
Scheduler: config values attempt and Cache error.

### ISSUE THREAD

**Issue by neoplomero** [antonioribeiro/health#129]
Hi guys. 

Nice intentions here, what really took my attention was the scheduler options so we can monitor apps on production.  But for this I found a few bugs:

1) on your ServiceProvider you're asking for:

`if (
            config('health.scheduler.enabled') &&
            ($frequency = config('health.scheduler.frequency')) &&
            config('health.notifications.enabled')
        )`

but on the published config file "schedule" is within "health.notifications" so your validations should be "health.notifications.scheduler..." instead.

Anyways after I change this to test it and finally put it to work. I started receiving a " Class cache does not exist" error related with a different function.  I don't have the time to keep digging here maybe if you find out the isue I'll be back to using it or maybe if I find a few hours on my week I can give you a hand with it. 

Thanks in advance.

**Comment by yehiaserag** [comment]
I'm having the same exact issue

**Comment by vlados** [comment]
I agree. I tried to dig into the issue but getting nowhere. Can someone tell us how to fix it or what to do to fix the package? 
**There is no use of the package if is not running in background!**
